core/templates/domain/collection/guest-collection-progress.service.ts

```diff
@@ -65,9 +63,13 @@ export class GuestCollectionProgressService {
   }
 
   getValidCompletedExplorationIds(collection: Collection): string[] {
-    var collectionId = collection.getId();
-    var guestCollectionProgress = this.loadGuestCollectionProgress();
-    var completedExplorationIds = (
+    const collectionId = collection.getId();
+    const guestCollectionProgress = this.loadGuestCollectionProgress();
+    let completedExplorationIds: string[];
```

Comment:
This line can be removed right? We can init the variable on line 72.

Reply:
done